apps/services/user-profile/src/app/v2/test/user-profile.spec.ts

```diff
@@ -0,0 +1,132 @@
+import request from 'supertest'
+
+import { createCurrentUser } from '@island.is/testing/fixtures'
+import { UserProfileScope } from '@island.is/auth/scopes'
+import { setupApp, setupAppWithoutAuth } from '@island.is/testing/nest'
+
+import { AppModule } from '../../app.module'
+import { SequelizeConfigService } from '../../sequelizeConfig.service'
+import { FixtureFactory } from './fixtureFactory'
+
+const testNationalId = '0101302399'
+
+const testUserProfile = {
+  nationalId: testNationalId,
+  email: 'test@test.is',
+  mobilePhoneNumber: '1234567',
+}
```

Comment:
Use the `createNationalId` and `createPhoneNumber` from `@island.is/testing/fixtures`. It is also nice to use the Faker library to create fake data, i.e. the email.
```suggestion
const testUserProfile = {
  nationalId: createNationalId(),
  email: faker.internet.email(),
  mobilePhoneNumber: createPhoneNumber(),
}
```